compliance/azure/instances_without_fnm_agent/README.md

```diff
@@ -0,0 +1,92 @@
+## Check for Azure instances not running FNMS agent
+
+### What it does
+This policy checks all instances running in Azure to determine if the FlexNet Inventory Agent is running on the instance and reports on any that are missing the agent.
+The policy is a report only policy, no action is taken during the Policy Escalation.
+
+### Functional Details
+The policy leverages the cloud API to get all current instances and the FlexNet Manager report (Custom view) API to get all azure cloud instances with agent. It cross-checks the two lists to determine if any instances are running on the cloud that aren't known to FNM.  The policy matches the ComputerName from FNMS and the VirtualMachine.name from Azure. 
+
+### Pre-reqs
+- FlexNet Manager
+- The following RightScale Credentials
+  - 'FNMS_API_Token'
+- Azure Service Principal (AKA Azure Active Directory Application) with the appropriate permissions to manage resources in the target subscription
+- The following RightScale Credentials
+  - `AZURE_APPLICATION_ID`
+  - `AZURE_APPLICATION_KEY`
+
+### Installation
+
+### How to retrive Azure `AZURE_APPLICATION_ID`, `AZURE_APPLICATION_KEY` and `Azure AD Tenant ID`
```

Comment:
This line is under Installation and should have 4 #### .

Reply:
Made the required changes. Thanks.